src/platforms/kotlin-multiplatform/native-access-sdk.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Native Access to SDK
+sidebar_order: 8
+description: "How to directly access the SDK without the shared module."
+---
+
+If you need to access the SDK directly in your native platform without the shared module, you first need to change the dependency configuration from `implementation(...)` to `api(...)`:
+
+```kotlin {filename:shared/build.gradle.kts}
+val commonMain by getting {
+  dependencies {
+    api("io.sentry:sentry-kotlin-multiplatform:{{@inject packages.version('sentry.kotlin.kmp', '0.0.1-alpha.2') }}")
+  }
+}
+```
+
```

Comment:
```suggestion

### Export the Framework
```

The sidebar gets confused if we don't follow markdown heading standards, so we need an H2 for these H3s